src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,28 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.NoSuchElementException;
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
```

Comment:
It seems like you're importing `CountryDaoImpl` directly, which is not a good practice. Instead, you should inject the `CountryDao` through the constructor, allowing for more flexible code that is easier to test and maintain. This way you can pass any implementation of `CountryDao` without changing the `CountryServiceImpl` class.